v5/pivotal/stories.go

```diff
@@ -155,6 +155,18 @@ type BlockerRequest struct {
 	Resolved    *bool  `json:"resolved,omitempty"`
 }
 
+// Review is a review on a PT story
+type Review struct {
+	ID           int        `json:"id,omitempty"`
+	StoryID      int        `json:"story_id,omitempty"`
+	ReviewTypeID int        `json:"review_type_id,omitempty"`
+	ReviewerID   int        `json:"reviewer_id,omitempty"`
+	Status       string     `json:"status,omitempty"`
```

Comment:
Would be nice to create `ReviewStatus` enum type and put it here so that people know what values are possible.

Reply:
I am happy to make this change, but looking at the existing API, I don't see any examples of types being used this way. Instead, the existing pattern seems to be to use a `string` but provide constants matching the possible values. Do you prefer the separate type or just strings with the appropriate constants?

Reply:
I misspoke. I do see two examples of types being used, `Day` and `AccountingType`. I'll pattern it off those.

Reply:
I've force pushed an updated version.